### ISSUE TITLE
make placing dairy farms on shrubland

### ISSUE THREAD

**Issue by Monsterfisch** [Sh0wdown/UnofficialCrusaderPatch#745]
just an idea to maybe make dairyfarms similary unique as they were in sh1 where they had the benefit of beeing placeable on areas where other farms were not able to be placed  (granted this will impact the balance of existent maps but as an option would provide players with new ways to play the game and create interesting szenarios.

mockup of the idea https://i.imgur.com/Sa6n1UY.jpg

**Comment by GRhin** [comment]
There is a way to do it currently, it involves placing grass in a specific pattern. The way it has been dont in the maps I have used is a little bugged, as placeing the farm extends grass a little, and so deleting the farm will allow the other farms to be used there. But there is another pattern.
It does not limit to only dairy, but limits to either dairy or apples. CIO has more info on that if you are interested.

**Comment by Monsterfisch** [comment]
yeah I know there are ways to force it but as you said it comes with certain issues for me the idea of making dairy farms more inline with their sh1 counterparts in terms of being able to be placed in areas other farms cannot be placed and shrubland just like in reality would support livestock as well. i am however open to looking at those patterns for future reference.

**Comment by Russianstory** [comment]
I support you furiously. I also wrote something similar somewhere for a long time. I agree! It would be great to use different types of grass in the game for different farms, or make all the farms available on different types of grass, BUT so that the "yield" is different depending on the suitability of the soil

**Comment by ByBurton** [comment]
Only question would be if the AI would know how to do that / use it.
